lib/game.rb

```diff
@@ -0,0 +1,136 @@
+class Game
+  attr_accessor :board, :player_1, :player_2, :winner
+
+    WIN_COMBINATIONS = [[0,1,2],
+                        [3,4,5],
+                        [6,7,8],
+                        [0,3,6],
+                        [1,4,7],
+                        [2,5,8],
+                        [0,4,8],
+                        [6,4,2]]
+  def initialize(player_1=nil, player_2=nil, board=nil)
+    @board = board || Board.new()
+    @player_1 = player_1 || Human.new("X")
+    @player_2 = player_2 || Human.new("O")
+    @winner = nil
+  end
+
+  def current_player
+    return @board.turn_count%2 == 0 ? @player_1 : @player_2
+  end
+
+  def over?
+    if won?
+      return true
+    elsif draw?
+      return true
```

Comment:
This was a copy/paste from the procedural TTT.  That one was a testing nightmare for me and this was how that ended up.  I like the look of `won? || draw?` for sure though.